devops/openshift/database/patroni-deploy.yml

```diff
@@ -31,10 +31,6 @@ objects:
       role: master
       cluster-name: ${NAME}
       app.kubernetes.io/name: patroni
-      sessionAffinity: None
```

Comment:
I notice that these configs were not present in the past either, nice catch.
Was it tested on DEV?